test/frontend/actions/siteActionsTest.js

```diff
@@ -149,15 +149,22 @@ describe('siteActions', () => {
       });
     });
 
-    it('triggers an error when adding a site fails', () => {
+    it('triggers an error and triggers a router update to the site list page when adding a site fails', () => {
       const siteToAdd = {
         hey: 'you',
       };
-      addSite.withArgs(siteToAdd).returns(rejectedWithErrorPromise);
+
+      // addSite returns nothing when the POST request fails, so resolve to undefined
+      addSite.withArgs(siteToAdd).returns(Promise.resolve(undefined));
```

Comment:
I didn't think you needed to pass undefined to Promise.resolve, does the test complain without an explicit value?

Reply:
you're right, will remove